README.md

```diff
@@ -52,11 +51,23 @@ Image can be used to visualize any tensor or intermediate generated image.
 <img src="https://raw.githubusercontent.com/daming-lu/large_files/master/loss_image.gif" width="60%"/>
 </p>
 
-### Histogram
-Histogram can be used to visualize parameter distribution and trends for any tensor.
+### Audio
+Audio can be used to play input audio samples or generated audio samples.
```

Comment:
generate

Reply:
This sentence means play 'generated' audio samples